Intersect.Server/Entities/Npc.cs

```diff
@@ -896,27 +895,43 @@ public override void Update(long timeMs)
                                 {
                                     case PathfinderResult.Success:
 
-                                        var dir = mPathFinder.GetMove();
-                                        if (dir > -1)
+                                        var dir = (Direction)mPathFinder.GetMove();
```

Comment:
GetMove is always returning a direction right? So the method should just return a Direction no cast needed here